src/Go/boardState/boardState.ts

```diff
@@ -59,6 +59,13 @@ export function getNewBoardState(
   return newBoardState;
 }
 
+export function getNewBoardStateFromSimpleBoard(
```

Comment:
You only use this once, I think it makes more sense to just do this where you use it instead of exporting a new function.

Reply:
I played around with this, but after the other changes it is fairly large, and also the "where you use it" is now inside the netscript implementation. Keeping this as a contained utility function here alongside similar ones ended up being cleaner.